### PR TITLE
Revisions to manual based on latest changes.

### PR DESCRIPTION
**Pull request by nigelhanlon**
Clubhouse story: https://app.clubhouse.io/condenastinternational/story/9894/update-flyway-manual

### CODE REVIEW

File: dashboard.md [hunk 1]
```diff
@@ -1,80 +1,64 @@
...
-
-[flyway-dashboard.prod.cni.digital](http://flyway-dashboard.prod.cni.digital)
-
-_Note: At the time of this writing the dashboard is still heavily in development, so documentation here is subject to change._
+The dashboard is where you'll start a migration from your source CMS to copilot. Each market place will be given a unique URL to access their own specific dashboard instance. This dashboard will be pre-configured with each markets known brands available for migration.
 
 ## Authentication
 
-The dashboard may require authentication to access it. The technical details of this has yet to be decided and as such cannot be documented right now.
+Each dashboard instance is protected by an IP whitelist which restricts access to know offices and networks. Market place offices should already be whitelisted as part of project kickoff, but additional IP's can be submitted for approval.
```
Comment (thekemkid) [inline dashboard.md]: "restricts access to know offices" - typo. know should be known

File: dashboard.md [hunk 2]
```diff
@@ -1,80 +1,64 @@
...
-* Integration API URL
-  The base URL of your integration API
-
-< Here we'll put some more config options when we decide on them >
-< We should put more detail on these options too (types, valid args, whether optional etc).
 
 ## Starting a migration
 
-When Flyway has been configured with your brand you will be able to start migrations through the dashboard.
+When Flyway has been configured with your brand you will be able to start migrations through the dashboard. When visiting the dashboard interface, you should be presented with the migration page by default. If not, it is available as a clickable link on the left menu.
```
Comment (thekemkid) [inline dashboard.md]: "When Flyway has been configured with your brand you will be able to start migrations through the dashboard." this can be ripped out

Comment (thekemkid) [inline dashboard.md]: "you should be presented" -> "you will be presented"

Comment (thekemkid) [reply dashboard.md]: "If not, it is available as a clickable link on the left menu." This can be taken out

File: dashboard.md [hunk 3]
```diff
@@ -1,80 +1,64 @@
...
 
 * Start a full migration.
   With this option, Flyway will begin iterating through your full paginated API to retrieve articles and migrate them.
 * Migrate specific articles.
   With this option you will need to provide a list of Integration API URLs to the dashboard to be migrated. When the migrate button is clicked, these articles will be migrated.
 
-## Stopping a migration
+_Please note:_
+- Full URLs are required when attempting either a full or specific article migration. 
+- The integration API brand **MUST** match the brand selected in the dashboard.
```
Comment (thekemkid) [inline dashboard.md]: "The integration API brand **MUST** match the brand selected in the dashboard." -> "The integration API brand code **MUST** match the brand code for the selected brand in the dashboard. e.g. the integration API should return the brand code `vit` when migrating for the `Vogue` brand in the Italian `it` market."

File: dashboard.md [hunk 4]
```diff
@@ -1,80 +1,64 @@
...
 
-When a migration is started, the required articles are added to the migration queue for processing by the Flyway service. In some cases you may wish to stop an ongoing migration, for example to reduce unexpected load on hosted services. To facilitate this, a 'Purge Queue' button is available on the Flyway Dashboard. Using the purge queue button, you can clear all currently queued article jobs which will stop any ongoing migration. You can monitor the progress of the purge request by watching the queue totals (detailed below);
+When a migration is started, the required articles are added to the migration queue for processing by the Flyway service. In some cases you may wish to stop an ongoing migration, for example to reduce unexpected load on hosted services. To facilitate this, a 'Purge Queue' button is available on the Migrate page of the Flyway Dashboard. Using the purge queue button, you can clear all currently queued article jobs which will stop any ongoing migration. You can monitor the progress of the purge request by watching the queue totals (detailed below);
```
Comment (thekemkid) [inline dashboard.md]: "In some cases you may wish to stop an ongoing migration, for example to reduce unexpected load on hosted services." -> "In some cases you may wish to stop an ongoing migration." Example unneeded, lets try be brief on this point.

File: dashboard.md [hunk 5]
```diff
@@ -1,80 +1,64 @@
...
 
 Before using the purge queue button, it is important to note the following:
 
 * It may take upwards of 10 minutes for a migration to stop as Flyway Service will finish migrating the currently running job.
 * All brands share the same migration queue. Purging will stop **all** ongoing migrations for **all brands** within this market.
-* After clicking the purge button, you must type *PURGE* in the confirmation dialog to continue. Clicking 'Cancel' or leaving the dialog box empty while clicking 'OK' will cancel the request.
+* After clicking the purge button, you must type the market name in the confirmation dialog to continue. Clicking 'Cancel' or leaving the dialog box empty while clicking 'OK' will cancel the request.
 * The migration queue can only be cleared once every 60 seconds. Multiple requests within this time period will succeed, but may be merged internally with the original request.
 * Attempting to clear the queue multiple times will not reduce the stopping latency of the migration.
+* Purging the migration queue currently does not affect the total failed items count.
```
Comment (thekemkid) [inline dashboard.md]: It is worth adding a note that if there is ongoing processing in flyway for a full migration, articles will continue to be queued until the API has finished being traversed.

We should add some way to stop this, as it is undesirable. cc/ @graycodes

File: flyway.md [hunk 6]
```diff
@@ -6,21 +6,22 @@ Flyway is a system for migrating content from your existing CMS into Copilot. Co
...
+2. **[Flyway Dashboard](/dashboard.md)**: This component is used to configure the _Flyway Service_ and start content migrations.
+3. **[Integration API](/integration.md)**: This component is to be developed as the interface that _Flyway Service_ communicates with to retrieve content from the Origin CMS. It is responsible for creating JSON which describes the content according to a set schema.
 
-The steps you would be expected to take to migrate content is as follows:
+The steps you would be expected to take to migrate content are as follows:
 
 * Develop an Integration API for a CMS of choice.
-* Go to the Flyway Dashboard and configure Flyway with the details of the Integration API.
-* Press the "Migrate" button on the Flyway Dashboard.
+* Go to the Flyway Dashboard and select the correct brand, then enter the full url of the Integration API.
```
Comment (thekemkid) [inline flyway.md]: "Go to the Flyway Dashboard and select the correct brand, then enter the full url of the Integration API." -> "Go to the Flyway Dashboard for your market, select the correct brand and then enter the full url of Integration API content to be migrated."

File: flyway.md [hunk 7]
```diff
@@ -6,21 +6,22 @@ Flyway is a system for migrating content from your existing CMS into Copilot. Co
...
+* Press the "Run Full Migration" button on the Flyway Dashboard.
```
Comment (thekemkid) [inline flyway.md]: "Press the "Run Full Migration" button on the Flyway Dashboard." -> "Press the "Run Full Migration" or "Migrate Articles" button on the Flyway Dashboard depending on which type of migration you wish to run, full or specific."

Comment (thekemkid) [reply flyway.md]: I actually think these bullet points could split into two - one set for full migration, a second for specific/partial migrations

File: dashboard.md (hunk 5 above)
Comment (graycodes) [reply dashboard.md]: Yeah, let's add that disclaimer. We'll fix the code later

File: flyway.md [hunk 8]
```diff
@@ -6,21 +6,22 @@ Flyway is a system for migrating content from your existing CMS into Copilot. Co
...
 
-The steps you would be expected to take to migrate content is as follows:
+The steps you would be expected to take to migrate content are as follows:
 
 * Develop an Integration API for a CMS of choice.
-* Go to the Flyway Dashboard and configure Flyway with the details of the Integration API.
-* Press the "Migrate" button on the Flyway Dashboard.
+* Go to the Flyway Dashboard for your market, select the correct brand and then enter the full url of Integration API content to be migrated.
+* Press the "Run Full Migration" or "Migrate Articles" button on the Flyway Dashboard depending on which type of migration you wish to run, full or specific.
 * Flyway Service will then iterate through and retrieve content from the Integration API.
```
Comment (thekemkid) [inline flyway.md]: This point doesn't make sense when migrating specific content

**Pull request merged**